package.json

```diff
@@ -49,6 +49,9 @@
 		"eslint": "^5.12.1",
 		"tape": "^4.9.2"
 	},
+	"files": [
```

Comment:
The files field is very dangerous and should never be used; only npmignore should be used to restrict what is published.

Reply:
@ljharb care to explain why it is so dangerous? npm [suggests to use it](https://blog.npmjs.org/post/165769683050/publishing-what-you-mean-to-publish) and there are some [bad stories](https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d) on using `.npmignore`.

Reply:
npm is wrong. The “bad stories” of npmignore is that one article of someone who ignored best practices and stored credentials within the filesystem of the repo - there’s tons of issues all over github of when someone has forgotten to include a file on “files” and broken their consumers. I’m quite sure.